modules/stitching/include/opencv2/stitching/detail/warpers.hpp

```diff
@@ -70,6 +70,16 @@ class CV_EXPORTS RotationWarper
      */
     virtual Point2f warpPoint(const Point2f &pt, InputArray K, InputArray R) = 0;
 
+    /** @brief Projects the image point backward.
+
+    @param pt Projected point
+    @param K Camera intrinsic parameters
+    @param R Camera rotation matrix
+    @return Backward-projected point
+    */
+    virtual Point2f warpPointBackward(const Point2f& pt, InputArray K, InputArray R) = 0;
```

Comment:
Please use this approach to bypass ABI check:

https://github.com/opencv/opencv/blob/1729297ef3bb87ea840c9ab7de9f97e03589fa02/modules/ml/include/opencv2/ml.hpp#L1300-L1304

Default implementation can contain `CV_Error(Error::StsNotImplemented, "");`